packages/eas-cli/src/utils/formatVariable.ts

```diff
@@ -0,0 +1,14 @@
+import dateFormat from 'dateformat';
+
+import formatFields from './formatFields';
+import { EnvironmentVariableFragment } from '../graphql/generated';
+
+export function formatVariable(variable: EnvironmentVariableFragment): string {
+  return formatFields([
+    { label: 'ID', value: variable.id },
+    { label: 'Name', value: variable.name },
+    { label: 'Value', value: variable.value || '' },
```

Comment:
```suggestion
    { label: 'Value', value: variable.value ?? '' },
```
nit: prefer `??` over `||`